src/qibo/backends/__init__.py

```diff
@@ -147,24 +181,32 @@ def create(self, dtype):
 
 
 def get_backend():
-    return str(GlobalBackend())
+    return str(_Global.get_backend())
 
 
 def set_backend(backend, **kwargs):
```

Comment:
It's true that this function is only an alias, but it is also the public alias, since the aliased one is private (by `_` convention).

Thus, it would deserve a docstring (that was not there even before this PR, but since you're upgrading it, it's a good time to add one).

Reply:
Added a docstring.